### DISCUSSION TITLE
i18n support is not compatible with next export

### DISCUSSION THREAD

**Discussion by zmeng0000** [vercel/next.js#65677] [Help]
### Summary

`i18n support is not compatible with next export`
`next build && next export`Static packaging reports this error. How can I solve it

**Comment by haydenbleasel** [comment]
Hi @zmeng0000, this is a known unsupported feature with the [Pages Router](https://nextjs.org/docs/pages/building-your-application/deploying/static-exports#unsupported-features) but it looks like it may have been addressed in the [App Router](https://nextjs.org/docs/app/building-your-application/deploying/static-exports#unsupported-features).
